snap/naming/tag.go

```diff
@@ -36,8 +36,8 @@ type ParsedSecurityTag interface {
 	InstanceName() string
 }
 
-// ParsedAppSecurityTag describes a parsed snap application security tag.
-type ParsedAppSecurityTag interface {
+// AppSecurityTag describes a parsed snap application security tag.
```

Comment:
s/describes/exposes the details of/

Reply:
which makes me think that maybe we should s/parsed/validated/ in these descriptions?